tests/main.c

```diff
@@ -5,49 +5,69 @@
 
 #include "test_helper.h"
 
-#include "mu_define_enum_2_without_invalid_test.h"
 #include "define_struct_test.h"
 #include "for_each_1_keep_2_test.h"
-#include "mu_eat_empty_args_test.h"
+#include "mu_convert_enum_test.h"
 #include "mu_count_array_items_test.h"
+#include "mu_count_1_or_more_arg_test.h"
+#include "mu_define_enum_2_test.h"
+#include "mu_define_enum_2_without_invalid_test.h"
 #include "mu_define_enum_test.h"
-#include "mu_convert_enum_test.h"
 #include "mu_define_local_enum_test.h"
-#include "mu_define_enum_2_test.h"
+#include "mu_eat_empty_args_test.h"
+#include "mu_is_empty_test.h"
+#include "mu_has_comma_test.h"
 #include "mu_pri_enum_test.h"
+#include "mu_the_nth_arg_test.h"
 
 int main(void)
 {
     int result;
-    result = run_mu_define_enum_2_without_invalid_test();
-    POOR_MANS_ASSERT(result == 0);
-    
-    result = run_mu_eat_empty_args_test();
-    POOR_MANS_ASSERT(result == 0);
-
-    result = run_define_struct_tests();
-    POOR_MANS_ASSERT(result == 0);
-
-    result = run_for_each_1_keep_2_tests();
-    POOR_MANS_ASSERT(result == 0);
-
-    result = run_mu_count_array_items_tests();
-    POOR_MANS_ASSERT(result == 0);
 
-    result = run_mu_define_enum_tests();
+    result = run_mu_the_nth_arg_tests();
     POOR_MANS_ASSERT(result == 0);
 
-    result = run_mu_convert_enum_tests();
+    result = run_mu_count_1_or_more_arg_tests();
     POOR_MANS_ASSERT(result == 0);
 
-    result = run_mu_define_local_enum_tests();
+    result = run_mu_has_comma_tests();
     POOR_MANS_ASSERT(result == 0);
 
-    result = run_mu_define_enum_2_tests();
+    result = run_mu_is_empty_tests();
     POOR_MANS_ASSERT(result == 0);
 
-    result = run_mu_pri_enum_tests();
-    POOR_MANS_ASSERT(result == 0);
+    //result = run_mu_define_enum_2_without_invalid_test();
+    //POOR_MANS_ASSERT(result == 0);
+    //
+    //result = run_mu_define_enum_2_without_invalid_test();
+    //POOR_MANS_ASSERT(result == 0);
+    //
+    //result = run_mu_eat_empty_args_test();
+    //POOR_MANS_ASSERT(result == 0);
+    //
+    //result = run_define_struct_tests();
+    //POOR_MANS_ASSERT(result == 0);
+    //
+    //result = run_for_each_1_keep_2_tests();
+    //POOR_MANS_ASSERT(result == 0);
+    //
+    //result = run_mu_count_array_items_tests();
+    //POOR_MANS_ASSERT(result == 0);
+    //
+    //result = run_mu_define_enum_tests();
+    //POOR_MANS_ASSERT(result == 0);
+    //
+    //result = run_mu_convert_enum_tests();
+    //POOR_MANS_ASSERT(result == 0);
+    //
+    //result = run_mu_define_local_enum_tests();
+    //POOR_MANS_ASSERT(result == 0);
+    //
+    //result = run_mu_define_enum_2_tests();
+    //POOR_MANS_ASSERT(result == 0);
+    //
+    //result = run_mu_pri_enum_tests();
+    //POOR_MANS_ASSERT(result == 0);
```

Comment:
Suspicion of these being erroneously commented

Reply:
Absolutely. They are now (iteration 6) uncommented. I believe you've looked over a draft, but even so, your comments are valid.

---
In reply to: [464138282](https://github.com/Azure/azure-macro-utils-c/pull/42#discussion_r464138282) [](ancestors = 464138282)